src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -89,13 +89,6 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
                     return new HttpResponseMessage(HttpStatusCode.Unauthorized);
                 }
 
-                // Validate the HttpMethod
```

Comment:
Realized that this code is now no longer needed after the recent http work, since route constraint validation is now done by the HttpRouteCollection automatically before we get to this point.